### ISSUE TITLE
Re-use `continuous_pickup/drop_off` instead of adding "LineString" geometry

### ISSUE THREAD

**Issue by gcamp** [MobilityData/gtfs-flex#53]
`continuous_pickup` is already a used field in GTFS, it might be worthwhile re-using more of the same concept instead of adding new geometry on `locations.geojson`. Is there some advantage to "LineString" that I'm missing?

**Comment by westontrillium** [comment]
+1 Agree that `continuous_pickup/drop_off` is suitable for flag stopping in between fixed stops as-is.

I wasn't aware that LineString geometry was intended for a flag stopping use case (is it? If not, what are the use cases?).

**Comment by gcamp** [comment]
> I wasn't aware that LineString geometry was intended for a flag stopping use case

That was my understanding from reading the spec. I'm not thinking of any other use case right now

**Comment by tsherlockcraig** [comment]
Agree it's generally duplicative. MD and I have discussed a few times over the years. Ultimately decided to push continuous_pickup and drop_off to adoption but also leave LineStrings as an option.

Arguably, there is a slightly different use case. If you're using continuous_pickup, you have to identify exactly when the trip starts with a departure time. If you provide a LineString geometry, you can represent a service that will depart in a window, but doesn't have a set time schedule.

**Comment by gcamp** [comment]
>Arguably, there is a slightly different use case. If you're using continuous_pickup, you have to identify exactly when the trip starts with a departure time. If you provide a LineString geometry, you can represent a service that will depart in a window, but doesn't have a set time schedule.

@tsherlockcraig wouldn't that difference not exist anymore after #55 ?

**Comment by gcamp** [comment]
I agree arbitrary limiting some type of geometry might seem weird but I don't like having multiple way of implementing the same thing. Potentially might be better to have LingString and remove `continuous_pickup` but that ship has sailed 🙃

**Comment by scmcca** [comment]
Whats missing here is the ability to have non-directional paths. 

`continuous_pickup` is useful for **directional** service where a vehicle is travelling in a fixed direction from one `stop_time` to the next. As [defined](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stop_timestxt):

> Indicates that the rider can board the transit vehicle at any point along the vehicle’s travel path as described by `shapes.txt`, from this `stop_time` to the next `stop_time` in the trip’s `stop_sequence`.

GTFS-Flex v2 allows non-directional service patterns within a single `stop_times.txt` record. The common case is by using a polygon geojson to describe a flexible service area. It seems like this concept extends at the linestring level to define specific **non-directional** paths of service.

 I might be missing something but I don't think that sort of thing is possible when you have a directional path using 2 or more entries in `stop_times.txt`.
